frontend/fingertips-frontend/playwright/testHelpers.ts

```diff
@@ -341,3 +354,36 @@ export function returnIndicatorIDsByIndicatorMode(
 export function sortAlphabetically(array: (string | null)[]) {
   array.sort((a, b) => a!.localeCompare(b!));
 }
+
+export const checkTrendsOnResultsPage = (): boolean => {
+  const checkTrends =
+    process.env.CHECK_RESULTS_TRENDS !== 'false' ? false : true;
```

Comment:
```suggestion
    process.env.CHECK_RESULTS_TRENDS === 'true';
```

Probably don't need the variable either.